src/test/unit/views/login-return.test.ts

```diff
@@ -0,0 +1,23 @@
+import request from 'supertest';
+import {app} from '../../../main/app';
+import {expect} from 'chai';
+
+const PAGE_URL = '/login/return';
+
+const navigationClass = 'moj-sub-navigation__link';
+const expectedHeader = 'Home';
+
+let htmlRes: Document;
+
+describe('Login return Page', () => {
+  beforeAll(async () => {
+    await request(app).get(PAGE_URL).then(res => {
+      htmlRes = new DOMParser().parseFromString(res.text, 'text/html');
+    });
+  });
+
+  it('should display header', () => {
+    const navigation = htmlRes.getElementsByClassName(navigationClass);
+    expect(navigation[0].innerHTML).contains(expectedHeader, 'Could not find the header');
```

Comment:
this test passes when it should fail now as youre missing the header which means its an empty page, so the test isnt actually testing what you think it is, it will need refactoring to make sure that it fails when the header isnt displayed and only passes when the header is displayed

Reply:
just in case its not clear:
this is what the page is rendering:
![Screenshot 2022-01-17 at 15 20 33](https://user-images.githubusercontent.com/26488566/149796343-1f0e32cd-81aa-4136-a194-8b2ff1038a16.png)

and this is your test passing when it should fail:

![Screenshot 2022-01-17 at 15 21 23](https://user-images.githubusercontent.com/26488566/149796416-afe577ce-5d3b-46c3-ace6-9902104a91ef.png)

Reply:
added the heading and added the test